README.md

```diff
@@ -22,25 +22,6 @@ This document is relevant to both editions, but if you have checked out this Cor
   Note that version 8.0 is currently not supported.
```

Comment:
Chii reported last night that MySQL 5.6 no longer works, so we should remove it from the list and state that 5.7 is the only version supported.  (And doing this will count as completing https://apromore.atlassian.net/browse/AP-3015)